src/models/helpers/flowControl.js

```diff
@@ -6,12 +6,13 @@
  * It can be either a string or an object with properties `name`, `prefix`, and `suffix`.
  * @returns A boolean indicating whether the function name matches the value.
  */
-function isMatch(
-  functionName: string,
-  value: string | { name: string, prefix?: boolean, suffix?: boolean },
-): boolean {
+function isMatch(functionName, value) {
+  //   functionName: string,
```

Comment:
Should this be commented?

Reply:
Yes. Garrett wrote this in Typescript, and I converted it to JavaScript as it was blocking scripts from being run the way it was. I commented stuff out instead of deleting because I assume Garrett will want to revisit when he returns.